app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -0,0 +1,117 @@
+      android:overScrollMode="never"
+      android:scrollbars="none">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/profile_chooser_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:gravity="end"
+          android:orientation="horizontal"
+          android:paddingTop="24dp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="parent">
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:minHeight="24dp"
+            android:text="@string/profile_chooser_language"
+            android:textColor="@color/profileChooserGreyTextColor"
+            android:textSize="16sp" />
+
+          <ImageView
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="20dp"
+            android:src="@drawable/ic_language_icon_grey_24dp" />
+        </LinearLayout>
+
+        <TextView
+          android:id="@+id/profile_select_text"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="36dp"
+          android:layout_marginTop="60dp"
+          android:layout_marginEnd="36dp"
+          android:fontFamily="sans-serif"
+          android:text="@string/profile_chooser_select"
+          android:textColor="@color/white"
+          android:textSize="24sp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.recyclerview.widget.RecyclerView
+          android:id="@+id/profile_recycler_view"
+          android:layout_width="match_parent"
+          android:layout_height="match_parent"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="36dp"
+          android:layout_marginEnd="16dp"
+          android:clipToPadding="false"
+          android:fadingEdge="horizontal"
+          android:fadingEdgeLength="72dp"
+          android:orientation="vertical"
+          android:overScrollMode="never"
+          android:paddingBottom="@dimen/bottom_white_space"
+          android:requiresFadingEdge="vertical"
+          android:scrollbars="none"
+          app:data="@{viewModel.profiles}"
+          app:layoutManager="androidx.recyclerview.widget.GridLayoutManager"
+          app:layout_constraintBottom_toBottomOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/profile_select_text"
+          app:spanCount="@integer/profilechooser_span"/>
+      </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.core.widget.NestedScrollView>
+
+    <LinearLayout
```

Comment:
Ditto here--given that the outer layout is a ConstraintLayout, prefer to lay out these views with constraints and avoid the extra layering.

Reply:
Done.